src/createServer.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const express = require('express');
+
+function createServer() {
+  const cors = require('cors');
+  const { router: usersRouter } = require('./routes/users.js');
+  const { router: expensesRouter } = require('./routes/expenses.js');
+  const expensesService = require('./services/expensesService.js');
+  const userService = require('./services/usersService.js');
+
+  expensesService.reset();
+  userService.reset();
```

Comment:
No need to reset, actually, we have a database here to keep the data, even if the app has stopped
```suggestion
```